Jenkinsfile

```diff
@@ -8,11 +8,8 @@ pipeline {
         timestamps()
     }
     parameters {
-        booleanParam(name: 'unit_validate', defaultValue: true, description: 'amd64 (x86_64) unit tests and vendor check')
+        booleanParam(name: 'validate', defaultValue: true, description: 'amd64 (x86_64) validate and vendor check')
         booleanParam(name: 'validate_force', defaultValue: false, description: 'force validation steps to be run, even if no changes were detected')
-        booleanParam(name: 'amd64', defaultValue: true, description: 'amd64 (x86_64) Build/Test')
-        booleanParam(name: 'rootless', defaultValue: true, description: 'amd64 (x86_64) Build/Test (Rootless mode)')
-        booleanParam(name: 'cgroup2', defaultValue: true, description: 'amd64 (x86_64) Build/Test (cgroup v2)')
         booleanParam(name: 'arm64', defaultValue: true, description: 'ARM (arm64) Build/Test')
         booleanParam(name: 's390x', defaultValue: false, description: 'IBM Z (s390x) Build/Test')
         booleanParam(name: 'ppc64le', defaultValue: false, description: 'PowerPC (ppc64le) Build/Test')
```

Comment:
The DCO check (below this line) can be moved to GHA or GitHub Apps https://github.com/apps/dco

Reply:
I think our DCO bot in enough for this validation;

![image](https://user-images.githubusercontent.com/1951866/187968782-c50c06ea-043b-4bf5-b471-4781d4ae5ec0.png)

But we could just remove the DCO script here. cc @thaJeztah

Reply:
I know we kept the DCO check in Jenkins as a bit of a "gate" to prevent spinning up machines on PRs that were missing a DCO. Maybe less relevant now that more things are removed, but perhaps better to discuss it separately.